compose/python/Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM python:3.6-alpine
-
-RUN apk update \
-  && apk add --virtual build-deps gcc python3-dev musl-dev file-dev \
-  && apk add curl 
+FROM dgladkov/snakepit:latest
```

Comment:
I would recommend continuing to use Docker alpine

Reply:
I prefer alpine likewise, but this image add python (2.7.17, 3.5.9, 3.6.9, 3.7.7, 3.8.2), 
and we can run tox with the diferents interpreter.